api/nodes/views.py

```diff
@@ -632,6 +632,21 @@ def get_queryset(self):
                 .values_list('child__pk', flat=True)
         return self.get_queryset_from_request().filter(pk__in=node_pks).can_view(auth.user).order_by('-modified')
 
+    def get_serializer_context(self):
+        context = super(NodeChildrenList, self).get_serializer_context()
+        region__id = self.request.query_params.get('region', None)
+        id = None
+        if region__id:
+            try:
+                id = Region.objects.get(_id=region__id).id
```

Comment:
Minor: This can be optimized slightly by only querying for the `id` column:

```python
id = Region.objects.filter(_id=region__id).values_list('id', flat=True).get()
````